stdlib/public/Platform/bionic.modulemap.gyb

```diff
@@ -282,6 +282,10 @@ module SwiftGlibc [system] {
     module sys {
       export *
 
+      module cdefs {
+        header "${GLIBC_ARCH_INCLUDE_PATH}/sys/cdefs.h"
+        export *
+      }
```

Comment:
Starting with Android 6, [features.h was emptied out and everything moved into sys/cdefs.h instead](https://android.googlesource.com/platform/bionic/+/f4c948a9e9b616c4f133f9495dacb105e51b3e49%5E%21/#F0). This is why [I had the earlier `__BEGIN_DECLS` and other ClangImporter issues with Foundation](https://github.com/apple/swift/pull/23208#issuecomment-516586826), which have since repeated with other modulemaps and are now fixed with this header.

Reply:
I would not mind accepting this patch as a standalone. If you create a new PR, mention me and I will accept.